### PR TITLE
Feature/angular13

### PR DESCRIPTION
**Pull request by mhenkens**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows our guidelines: https://github.com/NationalBankBelgium/ngx-form-errors/blob/master/CONTRIBUTING.md#-commit-message-guidelines
-   [x] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[x] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[x] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

run on angular 8

Issue Number: N/A

## What is the new behavior?

update the library from angular 8 to angular13 and add eslint as linting engine

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by SuperITMan — CHANGES_REQUESTED** [review]
As we discussed yesterday, could rebase your changes and leave the "ng7" and "ng8" applications alone until you have finished migrating the `ngx-form-errors` library to Angular 13 ? 
Then, upgrading both to ng12 and ng13 ?

File: .npmrc [hunk 1]
```diff
@@ -1 +1,4 @@
 registry=https://registry.npmjs.org
+
+; FIXME Set legacy-peer-deps=false and strict-peer-deps=true once upgraded Stark to Angular > 10
+legacy-peer-deps=true
```
[the remark this answers is not in the document]
Comment (mhenkens) [reply .npmrc:4]: yes is still needed

File: demo-app/ng12/test.json
[the remark this answers is not in the document]
Comment (mhenkens) [reply demo-app/ng12/test.json:1]: not needed removed

File: demo-app/ng13/test.json
[the remark this answers is not in the document]
Comment (mhenkens) [reply demo-app/ng13/test.json:1]: not needed removed

File: demo-app/ng12/package.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{
+  "name": "demo-app",
+  "version": "0.0.0",
+  "scripts": {
+    "clean": "npx rimraf ./dist ./reports",
+    "clean:modules": "npx rimraf ./node_modules package-lock.json",
+    "ng": "ng",
+    "start": "ng serve --disableHostCheck=true",
```
[the remark this answers is not in the document]
Comment (mhenkens) [reply demo-app/ng12/package.json:8]: work removed

File: demo-app/ng13/package.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
+{
+  "name": "demo-app",
+  "version": "0.0.0",
+  "scripts": {
+    "clean": "npx rimraf ./dist ./reports",
+    "clean:modules": "npx rimraf ./node_modules package-lock.json",
+    "ng": "ng",
+    "start": "ng serve --disableHostCheck=true",
```
[the remark this answers is not in the document]
Comment (mhenkens) [reply demo-app/ng13/package.json:8]: removed

**Comment by mhenkens** [comment]
> I was not able to run the ng12 demo app:
> 
> ```shell
> $ npm start                                                                                                   superitman@my-mac
> 
> > demo-app@0.0.0 start
> > ng serve --disableHostCheck=true
> 
> Warning: Running a server with --disable-host-check is a security risk. See https://medium.com/webpack/webpack-dev-server-middleware-security-issues-1489d950874a for more information.
> ⠹ Generating browser application bundles (phase: building)...node:internal/crypto/hash:69
>   this[kHandle] = new _Hash(algorithm, xofLen);
>                   ^
> 
> Error: error:0308010C:digital envelope routines::unsupported
> ```
> 
> Could you have a look ?

this issue is due to breaking change in node17 --> this work with node 14 but not node 18

see https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported

File: package.json [hunk 4]
```diff
@@ -21,9 +21,7 @@
     "url": "git+https://github.com/NationalBankBelgium/ngx-form-errors.git"
   },
   "keywords": [
-    "angular6",
-    "angular7",
-    "angular8",
+    "angular9",
```
Comment (SuperITMan) [inline package.json:24]: Could you change this to `"angular12", "angular13"`

**Pull request merged**